src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewPages.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
+import org.testng.Assert;
+
+public class SpecialNewPages extends WikiBasePageObject {
+  @FindBys(@FindBy(className = "mw-newpages-pagename"))
+  private List<WebElement> newArticleLinks;
+
+  public SpecialNewPages openSpecialNewPages(String wikiUrl) {
```

Comment:
maybe just call method `open` and istead of passing wikiUrl, you can just use urlBuilder to build url like here:
```  
public ArticlePageObject open() {
    getUrl(urlBuilder.getUrlForWiki(Configuration.getWikiName()) + URLsContent.WIKI_DIR
           + TestContext.getCurrentMethodName());

    return this;
  }
```

And there is no need to log this action, cause each navigation is logged by deafult using pre/post navigation hooks